.travis.yml

```diff
@@ -1,9 +1,30 @@
+version: ~> 1.0
 language: generic
-dist: bionic
+sudo: required
```

Comment:
I don't think this is required anymore to use any sudo commands, it can probably be removed